client/extensions/woocommerce/app/store-stats/referrers/index.js

```diff
@@ -166,30 +219,22 @@ class Referrers extends Component {
 					query={ query }
 					statType={ STAT_TYPE }
 				>
-					{ showSearch && (
-						<SearchCard
-							className={ 'referrers__search-filter' }
-							onSearch={ this.onSearch }
-							placeholder="Filter Referrers"
-							value={ filter }
-							initialValue={ filter }
+					<div className="referrers__widgets">
+						<StoreStatsReferrerConvWidget
+							fetchedData={ filteredSortedData }
+							unit={ unit }
+							siteId={ siteId }
+							query={ query }
+							statType={ STAT_TYPE }
+							endSelectedDate={ endSelectedDate }
+							queryParams={ queryParams }
+							slug={ slug }
+							limit={ LIMIT }
+							pageType="referrers"
+							selectedIndex={ selectedReferrerIndex }
+							selectedReferrer={ selectedReferrer.referrer }
 						/>
-					) }
-					<StoreStatsReferrerWidget
-						fetchedData={ filteredSortedData }
-						unit={ unit }
-						siteId={ siteId }
-						query={ query }
-						statType={ STAT_TYPE }
-						endSelectedDate={ endSelectedDate }
-						queryParams={ queryParams }
-						slug={ slug }
-						limit={ LIMIT }
-						pageType="referrers"
-						paginate
```

Comment:
I'm seeing the top 10 but when I select a referrer not in the top ten, its not showing up in the conversion widget, which I would have expected